compute/instances/troubleshooting/change-rescue-mode.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: Changing the rescue mode of your Instance
+  description: This page helps you to switch your Instance to the new rescue mode
+content:
+  h1: Changing the rescue mode of your Instance
+  paragraph: This page helps you to switch your Instance to the new rescue mode
+tags: rescue rescuemode bootscript boot-script instance boot script
+dates:
+  validation: 2023-05-04
+  posted: 2023-05-04
+categories:
+  - compute
+---
+
+The old, bootscript-based, rescue mode will be phased-out on May 31, 2023. To ensure the correct functioning of your Instances, please make sure to switch them to local boot or the new rescue mode before this date.
```

Comment:
The previous Instances rescue mode, which relied on bootscripts, will be gradually discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.